apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/ConsumerController.java

```diff
@@ -86,32 +74,33 @@ public ConsumerToken getConsumerTokenByAppId(@RequestParam String appId) {
     }
     if (Objects.equals("AppRole", type)) {
       return Collections.singletonList(consumerService.assignAppRoleToConsumer(token, appId));
-    }
-    if (StringUtils.isEmpty(namespaceName)) {
-      throw new BadRequestException("Params(NamespaceName) can not be empty.");
-    }
-    if (null != envs){
-      String[] envArray = envs.split(",");
-      List<String> envList = Lists.newArrayList();
-      // validate env parameter
-      for (String env : envArray) {
-        if (Strings.isNullOrEmpty(env)) {
-          continue;
+    } else {
```

Comment:
I think this `else {` statement is unnecessary

Reply:
From the git history. The code is merged from other branch.

Reply:
the `else {` statement was removed in pr #2864 , so I guess it was brought back to this branch when you resolved the merging conflicts.

Reply:
Change back already.

Reply:
Is the whole file rolled back? I think we still need to change the import statement `import com.ctrip.framework.apollo.portal.environment.Env;`?

Reply:
I've changed it by accident. A new commit has pushed.